src/quo2/components/colors/color/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns quo2.components.colors.color.view
+  (:require
+    [quo2.components.icon :as icon]
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.colors.color.style :as style]))
+
+(defn- feng-shui
+  [{:keys [color theme]}]
+  [rn/view
+   {:accessibile         true
+    :accessibility-label color
+    :style               (style/feng-shui theme)}
+   [rn/view {:style (style/left-half theme)}]
+   [rn/view {:style (style/right-half theme)}]])
+
+(defn- view-internal
+  [{:keys [color
+           secondary-color
```

Comment:
If I recall correctly, when I initially implemented the color-picker component, I used `secondary-color` to manage the `feng-shui` color. However, in this [PR](https://github.com/status-im/status-mobile/pull/16291/files#diff-8ff84bba0223e7da32cf67b4d2ea6b1a8f5aaac1c8e1f1df3dd3f70d36d334f1L19-L20) by @J-Son89, the feng-shui functionality was removed.

Based on my understanding of the new color implementation, it appears that we are now handling the feng-shui aspect differently. Consequently, I believe the `secondary-color` and its related code and logic are no longer necessary. @OmarBasem Could you please confirm?

Reply:
@ajayesivan `secondary-color` is not needed within the color picker for quo2. If `secondary-color` is not being needed in the component usages else where then we can remove it

Reply:
@ajayesivan after checking `secondary-color` it seems not to be needed. I have removed it 👍